onedrive_utils.py

```diff
@@ -0,0 +1,151 @@
+import sys
+import os
+from ast import literal_eval
+import yaml
+from datetime import timedelta, datetime
+
+# Has to be hardcoded because of non-unicode chars
+ONEDRIVE_FOLDER="D:\\OneDrive - TJ Sokol Královské Vinohrady\\HikLoad_commands"
```

Comment:
I guess it could be possible to add this folder as a command-line argument using the `argparse` library, by using the type `type=lambda s: unicode(s, 'utf8')`. Needs testing though.

Reply:
Actually my bad, Python 3 has all of the strings unicode. I just tested and `parser.add_argument("--folder", type=str)` works with unicode literals (It parsed `python onedrive_utils.py --folder "D:\\OneDrive - TJ Sokol Královské Vinohrady\\HikLoad_commands"` correctly)